packages/hardhat-plugin/src/index.ts

```diff
@@ -227,55 +232,106 @@ task("visualize")
     }
   );
 
-task("ignition-info")
-  .addParam("deploymentId")
-  .addFlag("json", "format as json")
```

Comment:
I have opted to remove the json option as status does more than just show the deployed addresses now.

Reply:
Agreed with this